config/development.yaml

```diff
@@ -81,5 +81,20 @@ kafka:
       brokers:
         - 127.0.0.1:9092
   topics:
-    visibility-topic:
+    cadence-visibility:
       cluster: test
+    cadence-visibility-dlq:
+      cluster: test
+  applications:
+    visibility:
+      topic: cadence-visibility
+      dlq-topic: cadence-visibility-dlq
+
+elasticsearch:
```

Comment:
Can we leave development config to use Cassandra as visibility store?

Reply:
Cassandra is always used. ES will become an additional store only when this config switched to enabled.